src/pages/Add.js

```diff
@@ -23,6 +28,19 @@ export default function Add() {
     };
 
     await postFormular(formular);
+    history.push('/Thanks');
+  }
+  const Div = styled.div`
```

Comment:
Excellent idea! I recommend renaming this component and putting it somewhere else so you can use it throughout your whole app

Reply:
Yeah, good idea :)